sdk/include/aditof/camera.h

```diff
@@ -463,6 +463,36 @@ class SDK_API Camera {
      */
     virtual Status adsd3500GetGenericTemplate(uint16_t reg,
                                               uint16_t &value) = 0;
+
+  /**
+   * @brief Read serial number from camera and update cache
+   * @return string
+   */
+  virtual aditof::Status readSerialNumber() = 0;
```

Comment:
4 methods for reading a serial number seem a bit too much. It would make sense to have something like this:
aditof::Status readSerialNumber(std::string &serialNumber, bool useCachedValue = true);